static/app/views/issueDetails/sampleEventAlert.tsx

```diff
@@ -22,9 +23,10 @@ export function SampleEventAlert({
         <LinkButton
           size="xs"
           priority="primary"
-          to={`/${organization.slug}/${project.slug}/getting-started/${
-            project.platform || ''
```

Comment:
I'm not sure why it was attempting to link to `/getting-started/<platform>/`, as far as I can tell that route does not exist